ace/validator.go

```diff
@@ -359,7 +371,7 @@ func validateAppAnnotations(metadataURL string, pm *schema.PodManifest, app *sch
 			continue
 		}
 
-		val, err := metadataGet(metadataURL, "/apps/"+string(app.Name)+"/annotations/"+key)
+		val, err := metadataGet(metadataURL, "/apps/"+string(app.Name)+"/annotations/"+key, "text/plain; charset=us-ascii")
```

Comment:
Not sure how we should represent this in the spec.  I didn't notice that this was possible from reading the spec originally. @eyakubovich